instantsearch.js/render-state/src/app.js

```diff
@@ -77,7 +77,7 @@ search.addWidgets([
   }),
   panel({
     templates: {
-      header: 'Brand',
+      header: () => 'Brand',
```

Comment:
not needed but fine

Reply:
Are strings still supported? I thought it only accepts a function now.